### PR TITLE
Add related tools to README

### PR DESCRIPTION
**Pull request by maelle**
1. I am not sure how the .md file was generated, I added a YAML header + put header.md in a separate file (like `pkgsearch`). I can set it back to whatever your workflow was. 
2. I'm not sure that ps cannot read from processes?

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -1,4 +1,14 @@
+---
+output:
+  md_document:
+    variant: markdown_github
+    toc: true
+    toc_depth: 4
+    includes:
+      before_body: header.md
+---
```
Comment (gaborcsardi) [inline README.Rmd]: Pls don't add this. The md is created like this:
https://github.com/r-lib/processx/blob/master/Makefile#L5

File: README.Rmd [hunk 2]
```diff
@@ -8,47 +18,6 @@ knitr::opts_chunk$set(
...
-         * [Standard output and error](#standard-output-and-error)
-         * [End of output](#end-of-output)
-         * [Polling the standard output and error](#polling-the-standard-output-and-error)
-         * [Polling multiple processes](#polling-multiple-processes)
-         * [Waiting on a process](#waiting-on-a-process)
-         * [Exit statuses](#exit-statuses)
-         * [Errors](#errors-1)
-   * [Code of Conduct](#code-of-conduct)
-   * [License](#license)
-
```
Comment (gaborcsardi) [inline README.Rmd]: Can you pls. put this back? Thanks.

File: README.Rmd [hunk 3]
```diff
@@ -414,6 +383,12 @@ p2$wait()
 p2$get_exit_status()
 ```
 
+## Related tools
+
+* The [`ps` package](http://ps.r-lib.org/) can query, list, manipulate all system processes, and processx uses it internally. The `ps` package _cannot_ create/launch system processes nor read from them. [Read more about `ps` in a blog post about `processx`](https://www.tidyverse.org/blog/2018/09/processx-3.2.0/#the-ps-package).
```
Comment (gaborcsardi) [inline README.Rmd]: IDK, I don't really like to state what a package _cannot_ do. It sounds like a deficiency. I would just remove the second sentence, and the blog post link does not belong here, either, imo.

Comment (maelle) [reply README.Rmd]: Ok! I was adding the "cannot" part to give a criterion for choosing one or the other.

File: README.Rmd (hunk 1 above)
Comment (gaborcsardi) [reply README.Rmd]: Can you call knitr manually, or just `make` from the command line? That'll update the `.md`.

File: README.Rmd (hunk 3 above)
Comment (gaborcsardi) [reply README.Rmd]: I think the source of the confusion is in the README / DESCRIPTION / etc. title: "Execute and control system processes from R". It should be "Execute and control subprocesses from R".

File: README.Rmd (hunk 1 above)
Comment (maelle) [reply README.Rmd]: Even if that makes my name appear everywhere?

File: README.Rmd (hunk 2 above)
Comment (maelle) [reply README.Rmd]: I now have, but why not use the automatic knitr TOC?

Comment (gaborcsardi) [reply README.Rmd]: because that didn't work when I created this README

File: README.Rmd (hunk 1 above)
Comment (gaborcsardi) [reply README.Rmd]: You don't have to push the irrelevant changes in the `.md` file.

File: README.Rmd (hunk 3 above)
Comment (maelle) [reply README.Rmd]: Should "not subprocesses" be a part of this sentence then?

**Pull request merged**